multiples.js

```diff
@@ -6,7 +6,14 @@
 exports.sumOfAMultiple = function( n ) {
   var sum = 0;
 
-  // do your work here
+  for(let i = 0; i < n; i++) {
+    if (i%3 === 0 & i%5===0){
```

Comment:
So the & here is not the doing a logical AND but rather a bitwise &. You can read through this link to learn about the differences between the two. https://stackoverflow.com/questions/7310109/whats-the-difference-between-and-in-javascript

Also, we really don't need to worry about checking to see the number is divisible by 15 since just checking against 3 OR 5 will give a truthy value and accept the number. It's basically the idea of lower common denominator.